src/platform/macos/events_loop.rs

```diff
@@ -544,6 +544,67 @@ impl Proxy {
     }
 }
 
+pub fn windows_key_code_for_char_code(c: char) -> Option<c_ushort> {
```

Comment:
```suggestion
pub fn char_to_keycode(c: char) -> Option<c_ushort> {
```

This function name made sense in Chromium, where they're explicit about their virtual keycodes being the same as what Windows has. While winit's `VirtualKeyCode` is also based on Windows keycodes, we 1) don't enforce that, and 2) most people don't know they're originally from Windows. So, a different function name would be easier to understand here. The name I entered in is just a suggestion, so feel free to go with something different.

Great job writing comments! For something inobvious like this, we usually also add [a link](https://github.com/WebKit/webkit/blob/82bae82cf0f329dbe21059ef0986c4e92fea4ba6/Source/WebCore/platform/cocoa/KeyEventCocoa.mm#L626) to whatever we referenced to learn how to implement it.

Reply:
Thanks for the reasoning.  How would you suggest I link to the reference?

Is
```
// Implementation reference: https://github.com/WebKit/webkit/blob/82bae82cf0f329dbe21059ef0986
    c4e92fea4ba6/Source/WebCore/platform/cocoa/KeyEventCocoa.mm#L626
```
enough?

Reply:
Yeah, that works! If in doubt, there's a bunch of examples: https://github.com/tomaka/winit/search?l=Rust&q=https